test/feature-benchmark/scenarios_scale.py

```diff
@@ -0,0 +1,77 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from textwrap import dedent
+
+from materialize.feature_benchmark.measurement_source import MeasurementSource, Td
+from materialize.feature_benchmark.scenario import Scenario
+
+
+class SmallClusters(Scenario):
+    """Materialized views across many small clusters."""
+
+    SCALE = 1.5  # 25 clusters
```

Comment:
30 or 25?

Reply:
Yes sorry, 32 in fact - 10 to the 1.5 power .